detox/src/configuration/loadExternalConfig.test.js

```diff
@@ -56,11 +57,39 @@ describe('loadExternalConfig', () => {
     expect(logger.warn).not.toHaveBeenCalled();
   });
 
+  it('should merge in the base config from "extends" property', async () => {
+    const { filepath, config } = await loadExternalConfig({ cwd: DIR_EXTENDS });
+
+    expect(filepath).toBe(path.join(DIR_EXTENDS, '.detoxrc.js'))
+    expect(config).toEqual({
+      extends: path.join(DIR_EXTENDS, 'middle.js'),
+      artifacts: {
+        rootDir: 'someRootDir',
+        plugins: {
+          log: "all",
+          screenshot: "all",
+          video: "all",
+        },
+      },
+    });
+  });
```

Comment:
I'm not well acquainted with this code so that may be the source of confusion, but I was sort of expecting a test that would `expect` the actual (post-merged) result... perhaps even a snapshot test. Is this not the place? (not referring to e2e's of course)